packages/backend-core/src/middleware/passport/local.js

```diff
@@ -55,6 +55,20 @@ exports.authenticate = async function (ctx, email, password, done) {
   if (await compare(password, dbUser.password)) {
     const sessionId = newid()
     const tenantId = getTenantId()
+
+    if (dbUser.provider || dbUser.providerType || dbUser.pictureUrl) {
+      delete dbUser.provider
+      delete dbUser.providerType
+      delete dbUser.pictureUrl
```

Comment:
We should go the other way and prevent certain users from setting / logging in using a password if they are a sso user. There are some tickets in the backlog to do this. This might be a risky change to make at the moment as it'll lock someone out from sso if they do log in with their existing password

Reply:
No problem. When I was shifting between providers and basic auth, I could see these elements were hanging around in the DB. Backing this out for now.